### DISCUSSION TITLE
Via对单个网站的Cookie清除好像无效

### DISCUSSION THREAD

**Discussion by srzx** [tuyafeng/Via#1570] [Ideas]
如题，举几个网站例子，比如weixin.qq.com。这个是清空了之后没有刷新，再去看还是有Cookies。

**Comment by zhangjing-GitHub-Code** [comment]
嘿，为什么没有半个人解决这个问题？？？？
这很重要的好吗
这是重大bug啊，起码说明一下是Google Webview还是Via的问题吧
**@srzx **
**@tuyafeng **

**Comment by zhangjing-GitHub-Code** [comment]
@solokot
@gregory678
@kae-sakura
@betterwebleon
麻烦解决一下，谢谢！
